docs/docs/04-standard-library/ex/redis.md

```diff
@@ -1,458 +0,0 @@
----
-title: Redis
```

Comment:
@Chriscbr - I believe the `Redis` winglib suppurts only the `sim` target, while the `ex.Redis` module supports both. I would sunset `ex.Redis` only after we migrate support for `tf-aws` and/or `awscdk` in the `Redis` winglib. See https://github.com/winglang/winglibs/blob/main/redis/redis.w

Reply:
Do we know of anyone using the Redis module in production right now? I understand the value in supporting users, but if the APIs aren't being unused I feel like it's better to avoid prolonging this cleanup.. the less code we have to maintain the faster we can ship.